dask/dataframe/tests/test_dataframe.py

```diff
@@ -1712,16 +1713,17 @@ def test_apply():
               df.apply(lambda xy: xy[0] + xy[1], axis='columns'))
 
     # inference
-    assert_eq(ddf.apply(lambda xy: xy[0] + xy[1], axis=1),
+    assert_eq(ddf.apply(lambda xy: xy[0] + xy[1], axis=1, meta=(None, int)),
```

Comment:
It seems that this part of the test is about inference, therefore calling `apply` without an explicit meta?